core-services/image-mirroring/knative/mapping_knative_v0_22_quay

```diff
@@ -0,0 +1,22 @@
+registry.ci.openshift.org/openshift/knative-v0.22.0:knative-eventing-test-event-flaker quay.io/openshift-knative/event-flaker:v0.22.0
+registry.ci.openshift.org/openshift/knative-v0.22.0:knative-eventing-test-event-flaker quay.io/openshift-knative/event-flaker:v0.22
```

Comment:
I checked, and this image is new, so I created a new repo here: https://quay.io/repository/openshift-knative/event-flaker?namespace=openshift-knative

and I added it to be updated (write) via our robot / bot account

Reply:
@lberk do you have access to our quay.io org?  If not, ping me offline